<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>index</title>

<script src="libs/header-attrs-2.14/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="libs/bootstrap-3.3.5/css/bootstrap.min.css" rel="stylesheet" />
<script src="libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="libs/navigation-1.1/tabsets.js"></script>
<link href="libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="libs/highlightjs-9.12.0/highlight.js"></script>
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-76258414-1', 'auto');
  ga('send', 'pageview');

</script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>






<link rel="stylesheet" href="includes/main.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">



<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <!-- NOTE: add "navbar-inverse" class for an alternate navbar background -->
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Computational Neurobiology Lab</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li><a href="index.html">Home</a></li>
        <li><a href="research.html">Research</a></li>
        <li><a href="publications.html">Publications</a></li>
        <li><a href="members.html">Members</a></li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">




</div>


<div id="computational-neurobiology-laboratory" class="section level1">
<h1>Computational Neurobiology Laboratory</h1>
<p>Led by Leon French at the Krembil Centre for Neuroinformatics, this
laboratory sought to understand diseases of the brain by mining
neuroscience resources for deeper insight into brain anatomy,
development and illness. Specifically, we focused on connectivity, gene
expression across the genome, brain, and body. We employed multifaceted
approaches that gain information by using data that crosses scales and
species.</p>
<div id="online-tools---mostly-all-are-prototypes"
class="section level3">
<h3>Online tools - mostly all are prototypes</h3>
</div>
<div id="human" class="section level2">
<h2>Human</h2>
<ul>
<li><a
href="https://poly-brain.shinyapps.io/polyprotein_drugs/">Polyprotein
compounds</a>: Drug mechanism of action polyprotein tester for the
proteome-wide atlas of drug mechanism of action from Mitchell et
al.</li>
<li><a
href="https://poly-brain.shinyapps.io/polyprotein_atlas/">Polyprotein
tissue (GTEx samples)</a>: Tissue specific polyprotein tester for the
quantitative proteome map of the human body from Jiang et al.</li>
<li>Polyprotein stats: Looks at simple and complex representations of
genes or proteins and tests if they can be used for classification
<ul>
<li><a href="https://probe-t5-protein-embeddings.streamlit.app/">ProtT5
embeddings</a></li>
<li><a href="https://polyproteinstats-czi.streamlit.app/">Cytoself
feature spectra</a></li>
<li><a
href="https://polyproteinstats-geneformer.streamlit.app/">Geneformer
embeddings from single cells</a></li>
<li><a href="https://probe-scgpt-gene-embeddings.streamlit.app/">scGPT
embeddings from single cells</a></li>
<li><a href="https://saprot-embeddings.streamlit.app/">SaProt structure
aware protein embeddings</a></li>
<li><a href="https://esm2-650m-embeddings.streamlit.app/">ESM2 650M
model protein embeddings</a></li>
<li><a href="https://prostt5-embeddings.streamlit.app/">ProstT5 protein
sequence and structure embeddings</a></li>
<li><a href="https://gene-pt-embeddings.streamlit.app/">GenePT gene
description based GPT-3.5 embeddings</a> and <a
href="https://gene-pt-embeddings-non-ncbi.streamlit.app/">without NCBI
descriptions</a> in prompt</li>
<li><a
href="https://nucleotide-transformer-embeddings.streamlit.app/">Nucleotide
transformer embeddings from 12k bases upstream of transcription start
sites</a></li>
<li><a href="https://nicheformer-embeddings.streamlit.app/">Nicheformer
embeddings</a></li>
<li><a href="https://apps.healthuniverse.com/elp-ilu-kfw/">Orthrus
embeddings</a></li>
<li><a href="https://hi-c-embed.streamlit.app/">Meta Hi-C PCA</a></li>
<li><a href="https://subcell-embed.streamlit.app/">SubCell fluorescence
microscopy embeddings</a></li>
<li><a href="https://esm-c-600-embeddings.streamlit.app/">ESM C 600M
embeddings</a></li>
<li><a href="https://cca-100-embed-tool.streamlit.app/">CCA 100 shared
dims on ProtT5, scGPT and Orthrus</a></li>
</ul></li>
<li><a
href="https://derekhoward.shinyapps.io/gtex_enrichment/">Polygenic
tissue</a>: Polygenic celltype tester for the bulk human GTEx
tissues</li>
<li><a
href="https://polygenic.shinyapps.io/polygenic_gtex_single_cell/">Single
cell tissue</a>: Polygenic celltype tester for the human GTEx single
cell pilot data from Eraslan et al.</li>
<li><a
href="https://derekhoward.shinyapps.io/enteric_enrichment/">Enteric
tester</a>: Polygenic tester for human enteric nervous system</li>
<li><a
href="https://polygenic.shinyapps.io/polygenic_singlecell_from_hpa/">Human
Protein Atlas tester</a>: Polygenic tester for Human Protein Atlas
single-cell data from Karlsson et al. (gene expression data)</li>
<li><a href="https://hbaset.msl.ubc.ca/">HBASets</a>: a tool for
multigene expression mapping in the human brain</li>
<li><a href="https://brain.shinyapps.io/polygenic_layers/">Developmental
zones</a>: Polygenic developmental layer tester for human and macaque
cortex</li>
<li><a
href="https://ethanhkim.shinyapps.io/transcriptome_app/">LaminaRGeneVis</a>:
Gene expression enrichment tester for human cortical layers</li>
<li><a
href="https://poly-brain.shinyapps.io/polygenic_snrna_from_siletti_et_al/">Adult
brain single nucleus data</a>: Polygenic tester for adult human brain
single nucleus dataset from Siletti et al.</li>
<li><a
href="https://poly-brain.shinyapps.io/polygenic_snrna_from_braun_et_al/">First-trimester
developing human brain single cell data</a>: Polygenic tester for fetal
human brain single cell dataset from Braun et al.</li>
<li><a
href="https://poly-brain.shinyapps.io/polygenic_organelles/">Global
organelle proteome profiling</a>: Polygenic tester for the global
organelle proteome profiling dataset from the Chan Zuckerberg
Biohub</li>
<li><a
href="https://polygenes.shinyapps.io/polygeneic_fetal_human/">Human
fetal atlas</a>: Polygenic tester for the human cell atlas of fetal gene
expression by Cao et al.</li>
<li><a
href="https://polygenes.shinyapps.io/polygeneic_white_adipose/">Human
white adipose tissue</a>: Polygenic tester for the abdominal white
adipose tissue cell atlas by Streets et al. </li>
<li><a
href="https://polygenes.shinyapps.io/polygenic_visceral_adipose/">Human
visceral adipose tissue</a>: Polygenic tester for the visceral adipose
tissue atlas by Lazarescu et al.</li>
<li><a
href="https://polygenes.shinyapps.io/polygeneic_blood_human/">Human
blood cells</a>: Polygenic tester for the reference blood cell atlas by
ScaleBio</li>
<li><a href="https://polygenes.shinyapps.io/polygenic_retina/">Human
retina atlas</a>: Polygenic tester for the human retina atlas by Jin Li
et al.</li>
<li><a
href="https://polyexpression.shinyapps.io/polygeneic_asian_immune/">Asian
Immune Diversity Atlas</a>: Polygenic tester for the Asian Immune
Diversity Atlas</li>
</ul>
</div>
<div id="monkey" class="section level2">
<h2>Monkey</h2>
<ul>
<li><a href="https://polygenic.shinyapps.io/nhpca_polygenic/">Polygenic
non human primate atlas</a>: Polygenic tester for the Cell
transcriptomic atlas of the non-human primate Macaca fascicularis data
(from Lei Han et al.)</li>
<li><a href="https://brain.shinyapps.io/polygenic_layers/">Polygenic
developmental zones</a>: Polygenic developmental layer tester for human
and macaque cortex</li>
</ul>
</div>
<div id="mouse" class="section level2">
<h2>Mouse</h2>
<ul>
<li><a href="https://brain.shinyapps.io/polygenic_celltypes/">Polygenic
celltypes</a>: Polygenic celltype tester for the mouse nervous
system</li>
<li><a
href="https://brain.shinyapps.io/polygenicsaunderssinglecell/">Polygenic
celltypes from Saunders et al. dataset</a>: Polygenic celltype tester
for the mouse nervous system</li>
</ul>
<div id="software-and-data" class="section level3">
<h3>Software and Data</h3>
<ul>
<li><a href="https://github.com/leonfrench">Leon’s GitHub page</a></li>
<li><a href="http://www.chibi.ubc.ca/WhiteText/">WhiteText Project</a>:
text mining for neuroanatomy</li>
<li><a
href="https://github.com/leonfrench/ABAMS/tree/master/BAMSandAllen">ABAMS
Project</a>: tools to analyze gene expression and connectivity in the
rodent brain</li>
<li><a
href="http://figshare.com/articles/A_FreeSurfer_view_of_the_cortical_transcriptome_generated_from_the_Allen_Human_Brain_Atlas/1439749">Freesurfer
view of the cortical transcriptome</a></li>
</ul>
</div>
<div id="relevant-local-mailing-lists-and-event-pages"
class="section level3">
<h3>Relevant local mailing lists and event pages</h3>
<ul>
<li><a
href="https://lists.torbug.org/mailman/listinfo/announce">TorBUG</a></li>
<li><a
href="http://www.neuroscience.utoronto.ca/aboutus/join-cpin.htm">Collaborative
Program in Neuroscience - CPIN</a></li>
<li><a href="https://stage.utoronto.ca/home/isss">Statistical methods
for genetics and genomics (not sure how to get on list)</a></li>
</ul>
</div>
</div>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
