---
output:
  html_document
---


# Computational Neurobiology Laboratory

Led by Leon French at the Krembil Centre for Neuroinformatics, this laboratory sought to understand diseases of the brain by mining neuroscience resources for deeper insight into brain anatomy, development and illness. Specifically, we focused on connectivity, gene expression across the genome, brain, and body. We employed multifaceted approaches that gain information by using data that crosses scales and species. 

### Online tools - mostly all are prototypes

## Human
* [Polyprotein compounds](https://poly-brain.shinyapps.io/polyprotein_drugs/): Drug mechanism of action polyprotein tester for the proteome-wide atlas of drug mechanism of action from Mitchell et al.
* [Polyprotein tissue (GTEx samples)](https://poly-brain.shinyapps.io/polyprotein_atlas/): Tissue specific polyprotein tester for the quantitative proteome map of the human body from Jiang et al.
* Polyprotein stats: Looks at simple and complex representations of genes or proteins and tests if they can be used for classification 
  * [ProtT5 embeddings](https://probe-t5-protein-embeddings.streamlit.app/) 
  * [Cytoself feature spectra](https://polyproteinstats-czi.streamlit.app/)
  * [Geneformer embeddings from single cells](https://polyproteinstats-geneformer.streamlit.app/)
  * [scGPT embeddings from single cells](https://probe-scgpt-gene-embeddings.streamlit.app/)
  * [SaProt structure aware protein embeddings](https://saprot-embeddings.streamlit.app/)
  * [ESM2 650M model protein embeddings](https://esm2-650m-embeddings.streamlit.app/)
  * [ProstT5 protein sequence and structure embeddings](https://prostt5-embeddings.streamlit.app/)
  * [GenePT gene description based GPT-3.5 embeddings](https://gene-pt-embeddings.streamlit.app/) and [without NCBI descriptions](https://gene-pt-embeddings-non-ncbi.streamlit.app/) in prompt
  * [Nucleotide transformer embeddings from 12k bases upstream of transcription start sites](https://nucleotide-transformer-embeddings.streamlit.app/)
  * [Nicheformer embeddings](https://nicheformer-embeddings.streamlit.app/)
  * [Orthrus embeddings](https://apps.healthuniverse.com/elp-ilu-kfw/)
  * [Meta Hi-C PCA](https://hi-c-embed.streamlit.app/)
  * [SubCell fluorescence microscopy embeddings](https://subcell-embed.streamlit.app/)
  * [ESM C 600M embeddings](https://esm-c-600-embeddings.streamlit.app/)
* [Polygenic tissue](https://derekhoward.shinyapps.io/gtex_enrichment/): Polygenic celltype tester for the bulk human GTEx tissues
* [Single cell tissue](https://polygenic.shinyapps.io/polygenic_gtex_single_cell/): Polygenic celltype tester for the human GTEx single cell pilot data from Eraslan et al.
* [Enteric tester](https://derekhoward.shinyapps.io/enteric_enrichment/): Polygenic tester for human enteric nervous system
* [Human Protein Atlas tester](https://polygenic.shinyapps.io/polygenic_singlecell_from_hpa/): Polygenic tester for Human Protein Atlas single-cell data from Karlsson et al. (gene expression data)
* [HBASets](https://hbaset.msl.ubc.ca/): a tool for multigene expression mapping in the human brain
* [Developmental zones](https://brain.shinyapps.io/polygenic_layers/): Polygenic developmental layer tester for human and macaque cortex
* [LaminaRGeneVis](https://ethanhkim.shinyapps.io/transcriptome_app/): Gene expression enrichment tester for human cortical layers
* [Adult brain single nucleus data](https://poly-brain.shinyapps.io/polygenic_snrna_from_siletti_et_al/): Polygenic tester for adult human brain single nucleus dataset from Siletti et al.
* [First-trimester developing human brain single cell data](https://poly-brain.shinyapps.io/polygenic_snrna_from_braun_et_al/): Polygenic tester for fetal human brain single cell dataset from Braun et al.
* [Global organelle proteome profiling](https://poly-brain.shinyapps.io/polygenic_organelles/): Polygenic tester for the global organelle proteome profiling dataset from the Chan Zuckerberg Biohub
* [Human fetal atlas](https://polygenes.shinyapps.io/polygeneic_fetal_human/): Polygenic tester for the human cell atlas of fetal gene expression by Cao et al.
* [Human white adipose tissue](https://polygenes.shinyapps.io/polygeneic_white_adipose/): Polygenic tester for the abdominal white adipose tissue cell atlas by Streets et al. 



## Monkey
* [Polygenic non human primate atlas](https://polygenic.shinyapps.io/nhpca_polygenic/): Polygenic tester for the Cell transcriptomic atlas of the non-human primate Macaca fascicularis data (from Lei Han et al.)
* [Polygenic developmental zones](https://brain.shinyapps.io/polygenic_layers/): Polygenic developmental layer tester for human and macaque cortex


## Mouse

* [Polygenic celltypes](https://brain.shinyapps.io/polygenic_celltypes/): Polygenic celltype tester for the mouse nervous system
* [Polygenic celltypes from Saunders et al. dataset](https://brain.shinyapps.io/polygenicsaunderssinglecell/): Polygenic celltype tester for the mouse nervous system

### Software and Data

* [Leon's GitHub page](https://github.com/leonfrench)
* [WhiteText Project](http://www.chibi.ubc.ca/WhiteText/): text mining for neuroanatomy
* [ABAMS Project](https://github.com/leonfrench/ABAMS/tree/master/BAMSandAllen): tools to analyze gene expression and connectivity in the rodent brain
* [Freesurfer view of the cortical transcriptome](http://figshare.com/articles/A_FreeSurfer_view_of_the_cortical_transcriptome_generated_from_the_Allen_Human_Brain_Atlas/1439749)

### Relevant local mailing lists and event pages

* [TorBUG](https://lists.torbug.org/mailman/listinfo/announce)
* [Collaborative Program in Neuroscience - CPIN](http://www.neuroscience.utoronto.ca/aboutus/join-cpin.htm) 
* [Statistical methods for genetics and genomics (not sure how to get on list)](https://stage.utoronto.ca/home/isss)
