<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>research</title>

<script src="libs/header-attrs-2.14/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="libs/bootstrap-3.3.5/css/bootstrap.min.css" rel="stylesheet" />
<script src="libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="libs/navigation-1.1/tabsets.js"></script>
<link href="libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="libs/highlightjs-9.12.0/highlight.js"></script>
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-76258414-1', 'auto');
  ga('send', 'pageview');

</script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>






<link rel="stylesheet" href="includes/main.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">



<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <!-- NOTE: add "navbar-inverse" class for an alternate navbar background -->
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Computational Neurobiology Lab</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li><a href="index.html">Home</a></li>
        <li><a href="research.html">Research</a></li>
        <li><a href="publications.html">Publications</a></li>
        <li><a href="members.html">Members</a></li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">




</div>


<div id="research-interests" class="section level2">
<h2>Research Interests</h2>
<p>Broadly, we are interested in helping integrate epigenomic,
transcriptomic, genetic, connectivity, imaging, clinical and
epidemiological data to understand the brain and gain insight into the
neurobiology of psychiatric disorders.</p>
<div id="integrative-neuroinformatics-for-neuroanatomy"
class="section level3">
<h3>Integrative Neuroinformatics for Neuroanatomy</h3>
<p>Over $400 million has been spent on the creation of comprehensive
gene expression, connectivity and methylation atlases of the mouse and
the human brain. In five years, more data will be available from the
Allen Institute for Brain Science, the Human Connectome Project, the
BRAIN Initiatives, and the European Human Brain Project. We create
methods and tools that will leverage this new data for fast analyses of
neuroanatomy. Our first aim is to provide maps of where (in the brain)
and when (during development) genes are expressed in the mouse and the
human brain. Currently, tools for polygenic expression analysis are not
available to scientists without advanced computational backgrounds. We
seek to complement the large and expensive gene expression atlases with
free online tools for polygenic mapping of the brain. Prototype versions
of these tools are <a href="./toolsAndData.html">available</a>. New
versions will include tests for cell type-specific patterns and
expression datasets from the UK Brain Expression Consortium and the NIH
Blueprint Non-Human Primate Atlas.</p>
</div>
<div id="region-and-layer-specific-gene-expression-markers"
class="section level3">
<h3>Region and layer specific gene expression markers</h3>
<p>We seek to characterize the genome across the major brain units of
cell type, cortical layer, and brain region. Several independent
datasets from mouse, macaque, and human have assayed gene expression
across these units. We plan to combine them to form an integrated
dataset that will be used to both model neuroanatomy and extract
biomarkers.</p>
</div>
<div id="polygenic-expression-mapping" class="section level3">
<h3>Polygenic expression mapping</h3>
<p>Decoding the genome has allowed us to discover associations between
specific genes and diseases, as well as to measure expression of all
genes in the genome in a specific tissue. Common mental illnesses are
complex, caused by many genes possibly interacting with various
environmental factors. Compounding this etiological complexity is the
heterogeneity of gene expression in the brain, with over 75% of genes
differentially expressed across brain regions or time. Many of the genes
associated with mental illness have unknown roles. While many scientists
are zooming in on these genes, they focus on specific brain regions at a
specific age for a single gene. We use computational approaches to
complement targeted efforts with whole brain and genome
perspectives.</p>
</div>
<div id="natural-language-processing" class="section level3">
<h3>Natural language processing</h3>
<p>We are interested in using written text to better characterize and
assess mental health and addiction. This work is supported by the
increasing amounts of our communications at that are archived online.
Inspiration is provided by work that has examined novels for markers of
cognitive decline <a
href="http://llc.oxfordjournals.org/content/26/4/435.full">(Le,
Lancashire, Hirst and Jokel, 2011)</a> and tweets of mothers for
postpartum language changes <a
href="http://research.microsoft.com/en-us/um/people/horvitz/predicting_postpartum_changes_chi_2013.pdf">(Choudhury,
Counts and Horvitz, 2013)</a>. We have participated in the <a
href="http://clpsych.org/shared-task-2019-2/">CLPsych</a> and <a
href="https://early.irlab.org/">eRisk</a> tasks that aim to triage and
predict signs of self-harm from social media posts.</p>
</div>
<div id="understanding-mr-images-from-the-transcriptomic-perspective"
class="section level3">
<h3>Understanding MR images from the transcriptomic perspective</h3>
<p>Integration of neuroimaging measurements with information from
genomic studies provides new molecular perspectives can reveal specific
tissue characteristics across neuroanatomy. Once brain images are
projected translated into genes it is possible to test for enriched
biological processes, cell type markers, and subcellular locations.
Brain region specific analyses can be performed to help determine if a
region’s neuroanatomical structure is defined by dynamic brain activity
or the slower scale of molecular transcription.</p>
</div>
<div id="parkinsons-disease" class="section level3">
<h3>Parkinson’s disease</h3>
<p>We seek to test for interactions between Parkinson’s associated genes
and the immune system in human gene expression and epigenetic resources.
This interest builds on past work that revealed differential
co-expression between the alpha-synuclein gene and interferon-gamma
signalling genes.</p>
</div>
</div>
<div id="collaborators" class="section level2">
<h2>Collaborators</h2>
<p>Current and recent past collaborators include:</p>
<ul>
<li>Drs. Tomáš Paus and Zdenka Pausova in Toronto (my postdoctoral
supervisors): population neuroscience, epigenetics and axon
transport</li>
<li>Drs. Franziska Wagner and Rüdiger W Veh in Germany: gene expression
analyses in the habenula</li>
<li>Dr. Noa Liscovitch in Israel: coexpression analyses of SNCA in
Parkinson’s disease</li>
<li>Dr. John B. Vincent in Toronto: expression analyses of genes
associated with autism and intellectual disability</li>
<li>Dr. Etienne Sibille in Toronto: neurobiology of depression and
aging</li>
<li>Dr. Paul Pavlidis in Vancouver (my PhD supervisor): bioinformatics
for neuroanatomical connectivity</li>
<li>Yohan Yee: text mining for Psychiatry</li>
</ul>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
